docs: modify filter doc (#283)
vanus-labs/docs: content/how-to/manage-subscription/filters.md

## Uses And Example

When creating a subscription, Use the optional parameter --filters.
See the following example:

```shell
vsctl subscription create \
  --name quick-start \
  --eventbus quick-start \
  --sink 'http://localhost:8080' \
  --filters '[
      { "exact": { "source": "vanus.source", "data.action": "created" } }
      { "prefix": { "type": "vanus.type" } }
    ]'
```

> Vanus filters are fully compatible with [CloudEvents filters][ce-filters]. It also supports the filtering of the data part when it is in a JSON format. The parameter 'filters' is an array of filter expressions. Each element is an object with only one key. Here are some filtering types.

Reviewer: ``` suggestion
Vanus filters are fully compatible with [CloudEvents filters][ce-filters]. Filters also support filtering of the data part when it is in a JSON format. The parameter 'filters' is an array of filter expressions, and each element is an object with only one key. Here are some filtering types.
```

> Vanus filters is fully compatible with [CloudEvents filters][ce-filters], and it's also extended to support the filtering of CloudEvents data that format must be JSON. The parameter filters is an array of filter expressions. Each element of the array is an object which only has one key which is filter types as following.

Reviewer: ```suggestion
Vanus filters are fully compatible with [CloudEvents filters][ce-filters]. It also supports the filtering of the data part when it is in a JSON format. The parameter 'filters' is an array of filter expressions. Each element is an object with only one key. Here are some filtering types.
```
